src/assets/simp/build/simp.spec

```diff
@@ -23,127 +23,128 @@ Obsoletes: pupmod-simp-activemq <= 4.0.1
 Obsoletes: pupmod-simp-mcollective <= 3.0.0
 
 # Core SIMP Requirements
-Requires: pupmod-camptocamp-kmod >= 2.1.0, pupmod-camptocamp-kmod < 3.0.0
+Requires: pupmod-camptocamp-kmod >= 2.2.0, pupmod-camptocamp-kmod < 3.0.0
 Requires: pupmod-simp-systemd >= 1.1.1, pupmod-simp-systemd < 2.0.0
 Requires: pupmod-herculesteam-augeasproviders_apache >= 3.0.0, pupmod-herculesteam-augeasproviders_apache < 4.0.0
-Requires: pupmod-herculesteam-augeasproviders_core >= 2.1.1-2016, pupmod-herculesteam-augeasproviders_core < 3.0.0
-Requires: pupmod-herculesteam-augeasproviders_grub >= 3.0.0, pupmod-herculesteam-augeasproviders_grub < 4.0.0
+Requires: pupmod-herculesteam-augeasproviders_core >= 2.1.4, pupmod-herculesteam-augeasproviders_core < 3.0.0
+Requires: pupmod-herculesteam-augeasproviders_grub >= 3.0.1, pupmod-herculesteam-augeasproviders_grub < 4.0.0
 Requires: pupmod-herculesteam-augeasproviders_postgresql >= 3.0.0, pupmod-herculesteam-augeasproviders_postgresql < 4.0.0
 Requires: pupmod-herculesteam-augeasproviders_puppet >= 2.1.1, pupmod-herculesteam-augeasproviders_puppet < 3.0.0
 Requires: pupmod-herculesteam-augeasproviders_shellvar >= 3.0.0, pupmod-herculesteam-augeasproviders_shellvar < 4.0.0
 Requires: pupmod-herculesteam-augeasproviders_ssh >= 3.0.0, pupmod-herculesteam-augeasproviders_ssh < 4.0.0
-Requires: pupmod-herculesteam-augeasproviders_sysctl >= 2.1.0-2016, pupmod-herculesteam-augeasproviders_sysctl < 3.0.0
+Requires: pupmod-herculesteam-augeasproviders_sysctl >= 2.2.0, pupmod-herculesteam-augeasproviders_sysctl < 3.0.0
 Requires: pupmod-onyxpoint-gpasswd >= 1.0.5, pupmod-onyxpoint-gpasswd < 2.0.0
 Requires: pupmod-puppet-yum >= 2.2.1, pupmod-puppet-yum < 3.0.0
-Requires: pupmod-puppetlabs-apache >= 1.10.0-2016, pupmod-puppetlabs-apache < 4.0.0
-Requires: pupmod-puppetlabs-concat >= 2.2.0-2016, pupmod-puppetlabs-concat < 5.0.0
+Requires: pupmod-puppetlabs-apache >= 3.0.0, pupmod-puppetlabs-apache < 4.0.0
+Requires: pupmod-puppetlabs-concat >= 4.1.1, pupmod-puppetlabs-concat < 5.0.0
 Requires: pupmod-puppetlabs-hocon >= 1.0.0, pupmod-puppetlabs-hocon < 2.0.0
-Requires: pupmod-puppetlabs-inifile >= 1.6.0-2016, pupmod-puppetlabs-inifile < 3.0.0
-Requires: pupmod-puppetlabs-java >= 1.2.0-2016, pupmod-puppetlabs-java < 3.0.0
+Requires: pupmod-puppetlabs-inifile >= 2.2.0, pupmod-puppetlabs-inifile < 3.0.0
+Requires: pupmod-puppetlabs-java >= 2.4.0, pupmod-puppetlabs-java < 3.0.0
 Requires: pupmod-puppetlabs-motd >= 1.9.0, pupmod-puppetlabs-motd < 2.0.0
 Requires: pupmod-puppetlabs-mount_providers >= 1.0.0, pupmod-puppetlabs-mount_providers < 2.0.0
```

Comment:
Required in 2 places:  simp and simp-extras.  Should be removed from simp-extras.